### PR TITLE
SEO-176062-xamarin-UG-page-with-redirects

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: Xamarin/Accordion/Accessibility.md [hunk 1]
```diff
@@ -35,4 +35,4 @@ The following image shows the AutomationIds of inner elements.
 
 ## See also
 
-[How to get an AccordionItem in Xamarin.Forms (SfAccordion)](https://www.syncfusion.com/kb/11439/)
\ No newline at end of file
+[How to get an AccordionItem in Xamarin.Forms (SfAccordion)](https://support.syncfusion.com/kb/article/9978/how-to-get-an-accordionitem-in-xamarin-forms-sfaccordion?isInternalRefresh=False)
```
Comment (Yvone-Atieno) [inline Xamarin/Accordion/Accessibility.md:38]: Check all KB links

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Accordion/Accessibility.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/Accordion/Accessibility.md]: resolved

File: Xamarin/Accordion/BindableLayout.md [hunk 2]
```diff
@@ -225,5 +225,5 @@ private void Accordion_Expanding(object sender, ExpandingAndCollapsingEventArgs
...
-[How to work with Accordion with SfListView in Xamarin.Forms (SfAccordion)](https://www.syncfusion.com/kb/11448/)`                                                                                                                                                             
+[How to bind ViewModel data to Accordion in Xamarin.Forms (SfAccordion)](https://support.syncfusion.com/kb/article/10004/how-to-bind-viewmodel-data-to-accordion-in-xamarin-forms-sfaccordion?isInternalRefresh=False)                                                                                                                                                                                                                              
+[How to work with Accordion with SfListView in Xamarin.Forms (SfAccordion)](https://support.syncfusion.com/kb/article/9966/how-to-work-with-accordion-with-sflistview-in-xamarin-forms-sfaccordion?isInternalRefresh=False)`
```
Comment (Yvone-Atieno) [inline Xamarin/Accordion/BindableLayout.md:229]: Check the above KB links and update properly

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**